src/Geta.404Handler/Core/Upgrade/Upgrader.cs

```diff
@@ -63,6 +64,19 @@ private static bool CreateRedirectsTable(DataAccessBaseEx dba)
             return created;
         }
 
+        private static bool AlterRedirectsTable(DataAccessBaseEx dba)
```

Comment:
This method adds `RedirectType`, so the name of the method should mention it - could be `AddRedirectTypeColumnToRedirectsTable`.
Also, update logger information accordingly.

Reply:
Corrected as per suggestion